cdk/src/functions/service-control-policies-root.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  AttachPolicyCommand,
+  CreatePolicyCommand,
+  DeletePolicyCommand,
+  DetachPolicyCommand,
+  ListRootsCommand,
+  OrganizationsClient,
+  PolicyType,
+  UpdatePolicyCommand,
+  ListPoliciesCommand,
+} from '@aws-sdk/client-organizations';
+import { CdkCustomResourceEvent, CdkCustomResourceResponse, Context } from 'aws-lambda';
+
+async function getRootId(organizationClient: OrganizationsClient): Promise<string | undefined> {
+  let id = 'error'; //set to error. Update if root ID is found. Then, update it to Root Id.
+  const command = new ListRootsCommand({});
+  const response = await organizationClient.send(command);
+
+  if (!response.Roots || response.Roots.length === 0) {
+    console.warn('No roots found in the organization');
+    return id;
+  }
+
+  const root = response.Roots[0];
+  id = root.Id || 'error';
+  return id;
+}
+
+async function getPolicyId(organizationClient: OrganizationsClient, policyName: string) {
+  const commandListPolicies = new ListPoliciesCommand({
+    Filter: PolicyType.SERVICE_CONTROL_POLICY,
+  });
+
+  const response = await organizationClient.send(commandListPolicies);
+
+  let policyId = 'error'; //set to error. Update if superwerker-root SCP policy is found. Then, update it to SCP Policy Id.
+
+  response.Policies?.forEach((policy) => {
+    if (policy.Name === policyName) {
+      policyId = policy.Id;
+    }
+  });
+
+  return policyId;
+}
+
+export async function handler(event: CdkCustomResourceEvent, _context: Context): Promise<CdkCustomResourceResponse> {
+  let client = new OrganizationsClient({ region: 'us-east-1' });
+
+  switch (event.RequestType) {
+    case 'Create':
```

Comment:
You are not enabling SCPs on organisation level. Previously there was a separate custom action for this. Or is this not required anymore?